dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -166,7 +166,7 @@
     @ApiModelProperty(value = "The id of the associated checker workflow")
     private BioWorkflow checkerWorkflow;
 
-    @ElementCollection(targetClass = Alias.class)
+    @ElementCollection(targetClass = Alias.class, fetch = FetchType.EAGER)
```

Comment:
This will eagerly fetch these across the board, leading to a bit of a performance hit. 
It would be preferable to just initialize/load these for the endpoint that you need them for, limiting the hit.